scripts/build.sh

```diff
@@ -93,6 +102,7 @@ docker build \
   --build-arg IMAGE_VCS_REF=${TRAVIS_COMMIT} \
```

Comment:
pls find a proper replacement for this variable as it is required to keep build information as part of the image (git commit id)

Reply:
You mean something like `git rev-parse --short HEAD` ?
I haven't included this, because in my oppinion the behaivior is the same as bevore, because TRAVIS_COMMIT isn't set somewhere inside this script, so if somebody called that script before, this was an empty variable.